android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -789,6 +829,26 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
         Log.d(TAG, text);
     }
 
+    private void setupSubtitlesButton() {
+        Log.d(TAG, "setupSubtitlesButton() " + player.getPlaybackState());
+        if (player != null && player.getPlaybackState() == ExoPlayer.STATE_READY) {
+
+
+            DcePlayerModel model = new DcePlayerModel(getContext(), player, trackSelector);
+
+            if (model.areSubtitlesAvailable() && model.areAudioTracksAvailable() && model.getAudioTracks().size() > 1) {
```

Comment:
why is the > 1 check needed? What if there are just multiple subs but single audio?

Reply:
It should be ||

Reply:
It should be ```if (model.areSubtitlesAvailable() || model.areAudioTracksAvailable() && model.getAudioTracks().size() > 1)```. There is > 1 because we don't want to show selection if there is only one audio track.